src/maven/maven.pro

```diff
@@ -55,7 +55,8 @@ FORMS = forms/settingsform.ui  \
         forms/selectadductsdialog.ui \
```

Comment:
`.pro` files are special files that work with `qmake` in `qt` projects to generate additional configuration and source files (along with a machine-generated `Makefile`).